inject-proxy/blockNavigation.js

```diff
@@ -8,22 +8,25 @@ const blockNavigation = () => {
   const { urlToProxy } = window.PDFTron;
 
   // block navigation for all a tags that don't start with #  
-  document.querySelectorAll('a:not([href^="#"])').forEach(elem => {
+  document.querySelectorAll('a:not([href]), a[href=""], a[href]:not([href^="#"]):not([href="javascript: void(0)"])').forEach(elem => {
```

Comment:
Please update the comment above to be more descriptive on the cases where we want to black navigation and add target blank.